.github/workflows/test-origami-component.yml

```diff
@@ -5,18 +5,14 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
This project is changing from listing itself as an Origami Component to instead be listed as a library.

We've [discussed on Slack](https://financialtimes.slack.com/archives/CSW6B2VAN/p1618828191093400) that non components should not use obt, let's change this workflow name and the steps to use `npm test` or something instead?

Reply:
I added a commit which swaps this to use it's own test runner and to use npm test instead of obt test for the github workflow -- [`10206a2` (#172)](https://github.com/Financial-Times/ftdomdelegate/pull/172/commits/10206a26e83972c937aeb6cf0c494bc6495e508c)